eProctoring/user/views.py

```
@@ -50,13 +51,14 @@ def register_view(request):
     return render(request, 'registration/register.html', {'form': form})
 
 
-
+@require_http_methods(["GET", "POST"])
 def logout_view(request):
     logout(request)
     messages.success(request, "You have successfully logged out.")
     return redirect('login')
 
 
+@require_http_methods(["GET", "POST"])
```

Ruff (0.8.2)

Undefined name `require_http_methods`

(F821)